codice/ddf
# DDF-4035: Add Policies and Enable Sec Manager for Standalone Solr

2.13.x port: https://github.com/codice/ddf/pull/3626

#### What does this PR do?
Enables the Pro-Grade security manager on the JVM running the standalone Solr instance.

#### Who is reviewing it? 
@mcalcote 
@ahoffer 

#### Select relevant component teams: 
@codice/security 

#### Ask 2 committers to review/merge the PR and tag them here.
@coyotesqrl 
@stustison

#### How should this be tested?
Run itests and verify that all Solr functionality such and ingest, query, and retrieval still work as expected.

#### What are the relevant tickets?
[DDF-4053](https://codice.atlassian.net/browse/DDF-4035)

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
